WORKSPACE

```diff
@@ -66,58 +89,39 @@ python_register_toolchains(
 # For xz linking
 # See https://qiita.com/ShotaMiyazaki94/items/d868855b379d797d605f
 
-_py_configure = """
-if [[ "$OSTYPE" == "darwin"* ]]; then
-    prefix=$(brew --prefix)
-    export LDFLAGS="-L$prefix/opt/xz/lib $LDFLAGS"
-    export CPPFLAGS="-I$prefix/opt/xz/include $CPPFLAGS"
-    export PKG_CONFIG_PATH="$prefix/opt/xz/lib/pkgconfig:$PKG_CONFIG_PATH"
-    ./configure --enable-shared --prefix=$(pwd)/bazel_install --with-openssl=$(brew --prefix openssl)
-else
-    ./configure --prefix=$(pwd)/bazel_install
-fi
-"""
-
-#http_archive(
-#    name = "python_interpreter",
-#    build_file_content = """
-#exports_files(["python_bin"])
-#filegroup(
-#    name = "files",
-#    srcs = glob(["bazel_install/**"], exclude = ["**/* *"]),
-#    visibility = ["//visibility:public"],
-#)
-#""",
-#    patch_cmds = [
-#        "mkdir $(pwd)/bazel_install",
-#        _py_configure,
-#        "make",
-#        "make install",
-#        "ln -s bazel_install/bin/python3 python_bin",
-#    ],
-#    sha256 = "0a8fbfb5287ebc3a13e9baf3d54e08fa06778ffeccf6311aef821bb3a6586cc8",
-#    strip_prefix = "Python-3.9.10",
-#    urls = ["https://www.python.org/ftp/python/3.9.10/Python-3.9.10.tar.xz"],
-#)
-
-#register_toolchains("//:sematic_py_toolchain")
-
-# Canonical interpreter
-load("@python3_8//:defs.bzl", "interpreter")
+# <add python version>: This section will need to be updated when a python version is added
```

Comment:
> `<add python version>`

Put this anywhere we'd need to update when adding a python version. E.g. python3.10